Sources/PackageLoading/TargetSourcesBuilder.swift

```diff
@@ -180,11 +180,6 @@ public struct TargetSourcesBuilder {
         diagnoseInfoPlistConflicts(in: resources)
         diagnoseInvalidResource(in: target.resources)
 
-        // It's an error to contain mixed language source files.
-        if sources.containsMixedLanguage {
-            throw Target.Error.mixedSources(targetPath)
-        }
```

Comment:
Pending investigation into Swift + Cxx package. This might be where we should replace this check with something like:
```swift
if sources.containsSwift && sources.containsCxx { ... }
```
```

Reply:
It seems that you can have a package with Swift and C++ together. You just don't get automatic interoperability and have to manually go through ObjC++